### PR TITLE
Fix test_resource_get and test_collection_get

### PR DESCRIPTION
**Pull request by egorhm**
Fix for test_resource_get and test_collection_get that are used for testing of resource GET calls.

### CODE REVIEW

File: test/integration/ggrc/services/test_common.py [hunk 1]
```diff
@@ -141,14 +145,22 @@ def test_missing_resource_get(self):
...
-  @SkipTest
   def test_collection_get(self):
+    """Test collection GET method from common.py"""
     date1 = datetime(2013, 4, 17, 0, 0, 0, 0)
     date2 = datetime(2013, 4, 20, 0, 0, 0, 0)
-    mock1 = self.mock_model(
-        modified_by_id=42, created_at=date1, updated_at=date1)
-    mock2 = self.mock_model(
-        modified_by_id=43, created_at=date2, updated_at=date2)
+    mock1 = self.mock_model(created_at=date1, updated_at=date1)
```
Comment (alieh-rymasheuski) [inline test/integration/ggrc/services/test_common.py]: It seems you don't use the generated mock model other than in `self.mock_json(mock model)`. Maybe it's better to make a method that generates the json itself?

It looks like this at the moment:
```python
  mock = self.mock_model(created_at, updated_at)
  json = self.mock_json(mock)
  self.assertSomething(json)
```
I'd prefer it this way:
```python
  json = self.mock_model_json(created_at, updated_at)
  self.assertSomething(json)
```

**Comment by alieh-rymasheuski** [comment]
@egorhm, looks good overall.

File: test/integration/ggrc/services/test_common.py (hunk 1 above)
Comment (egorhm) [inline test/integration/ggrc/services/test_common.py]: good notice. thanks!

**Comment by alieh-rymasheuski** [comment]
Approved.

**Pull request merged**